compliance/azure/azure_untagged_resources/untagged_resources.pt

```diff
@@ -97,20 +97,51 @@ datasource "ds_azure_resources" do
       field "region", jmes_path(col_item,"location")
       field "name", jmes_path(col_item,"name")
       field "tags", jmes_path(col_item,"tags")
+      field "type", jmes_path(col_item,"type")
     end
   end
 end
 
+# See also: https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/tag-support
+datasource "ds_tag_support" do
+  request do
+    host "raw.githubusercontent.com"
+    path "/tfitzmac/resource-capabilities/master/tag-support.csv"
```

Comment:
The csv file should probably live in the flexera/policy_template/data/azure path.  We do this for other things like regions and instance types.